envs/example/defaults.yml

```diff
@@ -107,8 +107,11 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+
 cinder:
-  enabled: True
+  enabled: "{{ ceph.enabled }}"
```

Comment:
Why is there a change to the cinder variable block that's based on ceph enablement?